app/src/main/java/com/mapbox/navigation/examples/standalone/custompuck/res/layout/mapbox_activity_custom_puck.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This `ConstraintLayout` is unnecessary, `MapView` can be a root view

Reply:
I updated it.